doc/conf.py

```diff
@@ -40,8 +40,8 @@
 master_doc = 'index'
 
 # General information about the project.
-project = u'resty'
-copyright = u'2012, Core Services Team'
```

Comment:
The `u'unicode'` syntax is valid for python 3.3+ (see: https://docs.python.org/3/whatsnew/3.3.html#summary-release-highlights) so there is no need to remove it.